ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    @objc public init(personaData: [PersonaData] = []) {
+        layout = ContactCollectionViewLayout()
+        layout.scrollDirection = .horizontal
+        super.init(frame: .zero, collectionViewLayout: layout)
+
+        if personaData.count > 0 {
+            contactList = personaData
+        }
+
+        configureCollectionView()
+        setupHeightConstraint()
+        register(ContactCollectionViewCell.self, forCellWithReuseIdentifier: ContactCollectionViewCell.identifier)
+        NotificationCenter.default.addObserver(self, selector: #selector(setupHeightConstraint), name: UIContentSizeCategory.didChangeNotification, object: nil)
+    }
+
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    // The array of PersonaData which is used to create each ContactView. The height constraint of the ContactCollectionView is updated when the count increases from 0 or decreases to 0.
```

Comment:
nit: use three "/" for method/class/constant documentation.

Example:
/// The array of PersonaData which is used to create each ContactView. 
/// The height constraint of the ContactCollectionView is updated when the count increases from 0 or decreases to 0.